subjects/devops/plus/README.md

```diff
@@ -0,0 +1,43 @@
+## plus
+
+### Instructions
+
+In this exercise, you will make a script that will take two arguments from the command line, add them and output the result.
```

Comment:
Give the exact name of the script here 
"Create a script `plus.sh` "

Reply:
Thanks. Done.